src/test/java/org/testng/reporters/jq/TimesPanelTest.java

```diff
@@ -0,0 +1,43 @@
+package org.testng.reporters.jq;
+
+import org.testng.ISuite;
+import org.testng.annotations.Test;
+import org.testng.internal.paramhandler.FakeSuite;
+import org.testng.reporters.XMLStringBuffer;
+import org.testng.xml.XmlClass;
+import org.testng.xml.XmlSuite;
+import org.testng.xml.XmlTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * Created by Oleg Shaburov on 28.10.2018
+ * shaburov.o.a@gmail.com
+ */
+public class TimesPanelTest {
+
+    private static final String GITHUB_1931 = "GITHUB-1931 [NPE] TimesPanel.maxTime(ISuite suite)";
+
+    @Test(description = GITHUB_1931)
+    public void generateReportTimesPanelContentForSuiteWithoutStartedTests() {
+        XmlSuite xmlSuite = new XmlSuite();
```

Comment:
If you extends SimpleBaseTest, you'll find some util methods for xml objects creation.